CHANGELOG.md

```diff
@@ -9,13 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Added attribute `geometry` to `Beam`.
 * Added `center_of_mass` property to Assembly class.
 * Added `volume` property to Assembly class.
 
 ### Changed
 
 * Reduced some boilerplate code in `Joint` subclasses.
 * Added argument `beams` to `Joint.__init__()` which expects tuple containing beams from implementing class instance.
+* Renamed `TimberAssembly` to `TimberModel`.
+* Renamed `compas_timber.assembly` to `compas_timber.model`.
+* Renamed `compas_timber.parts` to `compas_timber.elements`.
+* Based `Beam` on new `compas_model.elements.Element`.
+* Based `TimberModel` on new `compas_model.model.Model`.
+* Based `Joint` on new `compas_model.interactions.Interaction`.
```

Comment:
that seemed the most natural decision. you think different? in the end the analysis software down the line uses the interactions to know how the elements are physically interacting. It's still an open discussion what kind of properties an interaction has, but it seems logical the joint define it.

Reply:
I would assume that an `Interaction` is non-tangible, and a `Joint` is mostly a tangible thing, no?

Reply:
mmh yes there's definitely something about that. how would you model it differently? it's just that `Interaction` right now is literally empty, so hard to get a good feel of what it could be. According to @tomvanmele it could carry the semantics required by the analysis logic, but how it exactly will look like it not clear yet, I think.

Reply:
Looking at the code in `compas_model`, I guess the intention is to mean that "Interaction" can be any type of interaction, be it tangible or not. I'm not entirely convinced the word really conveys that meaning, and if `compas_model` aims to be the all-mighty model to rule them all, maybe we could review that, but it's not something for this PR actually,